File: TypeScript/백준/Bronze/2525. 오븐 시계/오븐 시계.ts
const fs = require("fs");
const input = fs.readFileSync("/dev/stdin").toString().trim().split("\n");

const [A, B] = input[0].split(" ").map(Number); // 현재 시각
const C = parseInt(input[1], 10);              // 요리 시간

const totalMinutes = A * 60 + B + C;
const hour = Math.floor((totalMinutes / 60) % 24); // 24시간제 처리
const minute = totalMinutes % 60;

console.log(hour, minute);

File: PyPy3/백준/Bronze/2525. 오븐 시계/오븐 시계.py
# 입력
A, B = map(int, input().split())  # 현재 시각
C = int(input())                  # 요리 시간

# 계산
total_minutes = A * 60 + B + C
hour = (total_minutes // 60) % 24  # 24시간제 처리
minute = total_minutes % 60

# 출력
print(hour, minute)
